uint8 id
bool move

